Explorer/Assets/DCL/Multiplayer/Connections/Archipelago/Rooms/Fixed/FixedConnectionRoomStrategy.cs

```diff
@@ -1,47 +1,39 @@
-using Cysharp.Threading.Tasks;
+﻿using Cysharp.Threading.Tasks;
 using DCL.Diagnostics;
-using DCL.Multiplayer.Connections.Archipelago.AdapterAddress.Current;
+using DCL.Multiplayer.Connections.Rooms;
 using DCL.Multiplayer.Connections.Rooms.Connective;
 using DCL.WebRequests;
-using LiveKit.Rooms;
 using System;
 using System.Threading;
 using UnityEngine;
 
 namespace DCL.Multiplayer.Connections.Archipelago.Rooms.Fixed
 {
-    public class FixedConnectiveRoom : IConnectiveRoom
+    /// <summary>
+    /// No need for any abstractions - it's a unique behaviour that we can't replace
+    /// </summary>
+    internal class FixedConnectionRoomStrategy : IRealmRoomStrategy
```

Comment:
Define please the "abstractions", something you are naming it and claiming it's bad to have many of it and something not

Reply:
Super simple interface:
- One level - no nesting
- One member `IConnectiveRoom ConnectiveRoom { get; }`, two implementations
- Clear purpose: to allow select polymorphic behavior between `Fixed` and `Archipelago` realms